nanoFramework.Json/TimeExtensions.cs

```diff
@@ -237,6 +311,18 @@ public static DateTime FromASPNetAjax(string ajax)
 		{
 			string[] parts = ajax.Split(new char[] { '(', ')' });
 
+			// If the input string is non-numeric, Convert.ToInt64() below will throw an exception from the Native code
+			// These exceptions cause annoying delays & output when running in the nanoFramework debugger
+			// Bail out if the input string is non-numeric
+			if (parts.Length < 2)
+			{
+				return DateTime.MaxValue;
+			}
```

Comment:
you need an extra enter after a closing } exception is for else if in this case

Reply:
Done.